docs/external_system.md

```diff
@@ -46,13 +46,13 @@ use externally-accessible API endpoints exposed by CSM.
    $ . ${SAT_VENV_PATH}/bin/activate
```

Comment:
Let's remove the `$ ` from the beginning of each line here, so the user can run these commands by copy-pasting them. In addition, at the beginning of the step, add a parenthesized indication of where the user should run this command. If we follow the docs-csm conventions, it would look something like this:

```
1. (`user@hostname>`) Create a Python virtual environment.
```

Reply:
Updated in next commit.